### ISSUE TITLE
PDFViewer: Tab navigation does not go beyond the Zoom level ComboBox in Safari on Mac

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#6235]
### Bug report
Tab navigation does not go beyond the Zoom level ComboBox in Safari on Mac

### Reproduction of the problem
* In Safari on Mac open the [PDFViewer Index demo](https://demos.telerik.com/kendo-ui/pdfviewer/index)
* Tab to navigate among the PDFViewer Toolbar buttons
* Note that after the Zoom level ComboBox the focus goes outside of the PDFViewer

### Expected/desired behavior
All buttons should be accessible upon navigation with tab.

### Environment
* **Kendo UI version:** 2020.3.1118
* **Browser:** [Safari on MAC]

**Comment by Dimitar-Goshev** [comment]
The above scenario is not relevant since R3 2022 - ToolBar navigation is enhanced in R3 2022 and navigation between PDFViewer tools is done via Arrow keys.
